source/Halibut/IHalibutRuntime.cs

```diff
@@ -45,7 +45,23 @@ public interface IHalibutRuntime : IDisposable
         /// TClientService would be: IClientFoo { void Bar(string, HalibutProxyRequestOptions); }
         /// </typeparam>
         /// <returns></returns>
-        public TClientService CreateClient<TService, TClientService>(ServiceEndPoint endpoint);
+        //public TClientService CreateClient<TService, TClientService>(ServiceEndPoint endpoint);
```

Comment:
Don't we need this for the way Tentacle currently passes the CancellationToken per request by using a IClientInterface?

Reply:
I'm surprised/confused we don't have tests for this in Halibut!!